src/config/Config.zig

```diff
@@ -4256,6 +4256,33 @@ pub const RepeatablePath = struct {
             var buf: [std.fs.max_path_bytes]u8 = undefined;
             const abs = dir.realpath(path, &buf) catch |err| abs: {
                 if (err == error.FileNotFound) {
+                    // Check if the path starts with a tilde and expand it to the home directory on linux/mac
+                    if (path[0] == '~') {
```

Comment:
This test needs to be a little more sophisticated, it should see if `path[1]` is the path separator because `~abc/` is the home directory of user `abc`. Only `~/` is the home directory of the current user.

Reply:
Updated this to now check for `~/`